sdk/devcenter/Azure.Developer.DevCenter/tests/Generated/Samples/Samples_DeploymentEnvironmentsClient.cs

```diff
@@ -0,0 +1,915 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Azure;
+using Azure.Core;
+using Azure.Developer.DevCenter;
+using Azure.Identity;
+using NUnit.Framework;
+
+namespace Azure.Developer.DevCenter.Samples
+{
+    public class Samples_DeploymentEnvironmentsClient
+    {
+        [Test]
+        [Ignore("Only validating compilation of examples")]
```

Comment:
I am assuming you do not run the samples for the same reason as tests running in playback mode?

Reply:
Yes.